bin/image_build_k8s_new_release

```diff
@@ -54,27 +50,18 @@ fi
 
 has_gh
 
-if [ -z "${REPO_PATH}" ]; then
-    REPO_PATH="${HOME}/go/src/github.com/rancher/image-build-kubernetes"
-fi
-
-if [ ! -d "${REPO_PATH}" ]; then
-    fatal "path does not exist: ${REPO_PATH}"
-fi
-
-cd "${REPO_PATH}"
-
 DATE=$(TZ=utc date '+%Y%m%d')
 
 for i in ${RELEASE_VERSIONS}; do
     TAG="${i}-build${DATE}"
 
     if [ ${DRY_RUN} ]; then
-        echo "gh release create ${TAG} --title ${TAG} --target master --notes ''"
+        echo "gh release create ${TAG} --repo rancher/image-build-kubernetes --title ${TAG} --target master --notes ''"
```

Comment:
Just asking. Is it worth to support repository override?

Reply:
For this script, no since this operation only applies to this repository. Same goes for the `release_rke2_rpms`. If the script is applicable against multiple repos, then absolutely.